pkg/plugin/plugin_test.go

```diff
@@ -291,6 +291,41 @@ func TestUninstall(t *testing.T) {
 	assert.NoFileExists(t, pluginDir)
 }
 
+func TestInformation(t *testing.T) {
+	pluginName := "test_plugin"
+
+	tempDir := t.TempDir()
+	pluginDir := filepath.Join(tempDir, ".trivy", "plugins", pluginName)
+
+	os.Setenv("XDG_DATA_HOME", tempDir)
+	defer os.Unsetenv("XDG_DATA_HOME")
```

Comment:
We can use Setenv after Go 1.17.
https://pkg.go.dev/testing#T.Setenv

```suggestion
	t.Setenv("XDG_DATA_HOME", tempDir)
```